rdm/init_files/documents/software_plan.md

```diff
@@ -467,18 +491,20 @@ When creating a new problem report, include in the description:
-2. Evaluate the problem's relevance to safety using the software [risk assessment activity](#risk-assessment) [[62304:6.2.1.3]]
-3. Consider whether the software items implicated in the investigation have the proper safety class, and address as appropriate [[62304:6.2.2]]
-4. Summarize the conclusions from the investigation in the problem report
-5. Create a change request for actions needed to correct the problem (also include an issue reference to the problem report [[62304:8.2.4.a and 62304:8.2.4.b]], or document the rationale for taking no action [[62304:9.2]].
+1. Investigate the problem and if possible identify the cause and record it in the problem report [[62304:9.2.a]]
+2. Evaluate the problem's relevance to safety using the software [risk assessment activity](#risk-assessment) [[62304:6.2.1.3 62304:9.2.b]]
+3. Consider whether the software items implicated in the investigation have the proper safety class, and address as appropriate [[62304:6.2.2 62304:7.2.2.b]]
+4. Summarize the conclusions from the investigation in the problem report [[62304:9.2.c]]
+5. Create a change request for actions needed to correct the problem (also include an issue reference to the problem report [[62304:8.2.4.a and 62304:8.2.4.b 62304:9.2.d]], or document the rationale for taking no action [[62304:9.2.d]].
 
-**If the problem affects devices that have been released, make sure that quality control is aware of the situation and has enough information to decide whether and how to notify affected parties, including users and regulators.  Record who you notified in the problem report [[62304:9.3 and 62304:6.2.5; this document does not specify the process by which quality assurance will inform users, when they must inform users, etc.  It is assumed these details are handled in another process, and that all that the software developers must do is pass along the appropriate details to quality assurance.]].**
+**If the problem affects devices that have been released, make sure that quality control is aware of the situation and has enough information to decide whether and how to notify affected parties, including users and regulators.  Record who you notified in the problem report [[62304:9.3 62304:6.2.5.a and 62304:6.2.5.b; this document does not specify the process by which quality assurance will inform users, when they must inform users, etc.  It is assumed these details are handled in another process, and that all that the software developers must do is pass along the appropriate details to quality assurance.]].**
 
 **Output:** Details about the problem investigation documented in the problem report and either unapproved change requests or justification as to why change requests weren't necessary
 
 **Verification:** Not applicable to this activity
 
+{# TODO: Add an activity concerning gathering of production and post-production information for 14971:3.4.f and relevant portion of 14971:3.1 #}
```

Comment:
Perhaps this could be included in the "Release Planning" activity, alongside analyzing historical problem reports. See:

https://github.com/innolitics/rdm/pull/54/files#diff-e09d4b5bb67e18d545cd896d8d701743R282

Reply:
The release planning output is a set of change requests. I am thinking **using** production and post production information fits there but "**gathering** of..." needs to be its own activity. For this PR I am leaving it be.

Reply:
good point; leaving as is makes sense to me